src/sentry/profiles/task.py

```diff
@@ -131,6 +131,8 @@ def process_profile_task(
         set_measurement("profile.samples", len(profile["profile"]["samples"]))
         set_measurement("profile.stacks", len(profile["profile"]["stacks"]))
         set_measurement("profile.frames", len(profile["profile"]["frames"]))
+    elif "profiler_id" in profile and profile["platform"] == "android":
+        sentry_sdk.set_tag("format", "chunk")
```

Comment:
Should be more specific and say android chunk? For those who don't know this feels like it includes all chunks